### PR TITLE
feat: removing summaryhook-aside which now lives in the ai-aside repo

### PR DESCRIPTION
**Pull request by ashultz0**
This removes the xblock requirement so a make upgrade is included to pick that up.

edx-arch-experiments is no longer installed on edx.org

**Merge checklist:**
Check off if complete *or* not applicable:
- [x] Version bumped
- [x] Changelog record added
- [x] Documentation updated (not only docstrings)
- [x] Fixup commits are squashed away
- [x] Unit tests added/updated
- [x] Manual testing instructions provided
- [x] Noted any: Concerns, dependencies, migration issues, deadlines, tickets

### CODE REVIEW

File: requirements/test.txt [hunk 1]
```diff
@@ -1,13 +1,9 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.9
```
Comment (timmc-edx) [inline requirements/test.txt:2]: We should be using Python 3.8 virtualenvs for all `make upgrade` jobs to match the server environment.

File: edx_arch_experiments/__init__.py [hunk 2]
```diff
@@ -2,4 +2,4 @@
 A plugin to include applications under development by the architecture team at 2U.
 """
 
-__version__ = '1.2.0'
+__version__ = '1.3.0'
```
Comment (timmc-edx) [inline edx_arch_experiments/__init__.py:5]: Could you bump this to 2.0.0 to indicate the breaking change?

File: requirements/test.txt (hunk 1 above)
Comment (ashultz0) [reply requirements/test.txt]: rebased into 002b2ce

File: edx_arch_experiments/__init__.py (hunk 2 above)
Comment (ashultz0) [reply edx_arch_experiments/__init__.py]: good call, rebased into 002b2ce

**Review by timmc-edx — APPROVED** [review]

**Pull request merged**